script/openqa-clone-job

```diff
@@ -21,111 +21,114 @@
 
 =head1 openqa-clone-job
 
-openqa-clone-job - clone job from local or remote openQA instances
+Clones a job from the local or another openQA instance. Downloads all assets associated
+with the job (unless --skip-download is specified). Optionally settings can be modified.
 
 =head1 SYNOPSIS
 
   openqa-clone-job [OPTIONS] JOBREF [KEY=[VALUE] ...]
 
+  # clones job 42 (and parents) from "openqa.opensuse.org" to the local openQA instance
```

Comment:
I wonder if this is a common format for SYNOPSIS in perl applications. Have you seen this elsewhere?

Also I would write "(and any existing parents)"

Reply:
> Have you seen this elsewhere?

I don't know. I just find it annoying to have "nice" examples but one still has to figure out what these examples actually do.

Reply:
It looks fine to me, I don't see a problem

Reply:
ok, thanks for the confirmation. Keep it then.